config/config.go

```diff
@@ -320,6 +320,8 @@ type Log struct {
 	ExpensiveThreshold  uint   `toml:"expensive-threshold" json:"expensive-threshold"`
 	QueryLogMaxLen      uint64 `toml:"query-log-max-len" json:"query-log-max-len"`
 	RecordPlanInSlowLog uint32 `toml:"record-plan-in-slow-log" json:"record-plan-in-slow-log"`
+	AuditLogFile        string `toml:"audit-log-file" json:"audit-log-file"`
```

Comment:
I'm not sure if this is good to add those params in Log sturct. Or I can have a new type called AuditLog which includes `Filename`, `Level` and so on. @bb7133

Reply:
BTW, it's same as `SlowQueryFile` and `SlowThreshold`. I'm not sure which one is better.